user_guide/configuration/feature_suite_and_scenario_configuration.rst

```diff
@@ -160,6 +160,48 @@ be tested in ``UserContext`` and features described from the
 perspective of the *admin* actor will be tested in ``AdminContext``.
 Even if they are in the same folder.
 
+While it is possible to specify filters as part of suite configuration,
+sometimes you will want to exclude certain scenarios across the suite, with the
+option to override the filters at the command line.
+
+This is achieved by specifying the filter in the gherkin configuration:
+
+.. code-block:: yaml
+
+    # behat.yml
+
+    default:
+        gherkin:
+          filters:
+            tags: ~@wip
+
+In this instance, scenarios tagged as @wip will be ignored unless the CLI
+command is run with a custom filter, e.g.:
+
+.. code-block:: bash
+
+    vendor/bin/behat --tags=wip
+
+.. tip::
+
+   More details on identifying tests can be found in the chapter
+   :doc:`/user_guide/command_line_tool/identifying_tests`.
+
+Extensions
```

Comment:
This chapter is about configuring suites. Mixing other configurations in the same chapter would be confusing.

And the chapter about configuring extension will stay small anyway, as this is mostly about how to register an extension. Specific configuration will be described in the doc of each extension